cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/snowbridge.rs

```diff
@@ -135,6 +135,32 @@ fn ethereum_to_polkadot_message_extrinsics_work() {
 	);
 }
 
+/// Tests that the digest items are as expected when a Ethereum Outbound message is received.
+/// If the MessageQueue pallet is configured before (i.e. the MessageQueue pallet is listed before
+/// the EthereumOutboundQueue in the construct_runtime macro) the EthereumOutboundQueue, this test
+/// will fail.
+#[test]
+pub fn test_pallet_order_works() {
```

Comment:
@claravanstaden I would suggest to rename this `test_pallet_order_works` to something very specific - exactly what it does, e.g. `fn ethereum_outbound_queue_processes_messages_before_message_queue_works` or something more descriptive

Reply:
Agreed, done in https://github.com/paritytech/polkadot-sdk/pull/3381/commits/e7fc3a42b426cb87c0dd8a0f053fb1469de5ed3c.